apps/admin/backend/src/app.test.ts

```diff
@@ -232,15 +233,6 @@ test('setSystemSettings returns an error for malformed input', async () => {
   assert(result.isErr());
   const err = result.err();
   expect(err.type).toEqual('parsing');
-  expect(JSON.parse(err.message)).toMatchObject([
-    {
-      code: 'invalid_type',
-      expected: 'boolean',
-      received: 'undefined',
-      path: ['arePollWorkerCardPinsEnabled'],
-      message: 'Required',
-    },
-  ]);
```

Comment:
This is still a part of the error message but so are entries for all the other new system settings fields.

So that we don't have to update this every time we add a new system setting, I think the `expect(err.type).toEqual('parsing');` check is probably sufficient